flink-examples/flink-examples-table/src/main/java/org/apache/flink/table/examples/java/StreamSQLExample.java

```diff
@@ -69,7 +69,7 @@ public static void main(String[] args) throws Exception {
 			tEnv = StreamTableEnvironment.create(env, settings);
 		} else {
 			System.err.println("The planner is incorrect. Please run 'StreamSQLExample --planner <planner>', " +
-				"where planner (it is either flink or blink, and the default is flink) indicates whether the " +
```

Comment:
Could you also update the class StreamSQLExample.scala?

Reply:
Thanks for your reminder.
I actually ignored it. 
I will submit this update later.